freqtrade/optimize/backtesting.py

```diff
@@ -398,6 +399,10 @@ def start(self) -> None:
                 'Backtesting with data from %s up to %s (%s days)..',
                 min_date.isoformat(), max_date.isoformat(), (max_date - min_date).days
             )
+
+            # Store reprocessed data for later use in export
+            preprocessed_data[self.strategy.get_strategy_name()] = preprocessed
```

Comment:
You might wanna extract the "datadump" function from after the loop into here.

Assuming this is running for 20 strategies, you'd force me to be able to have enough RAM available to store the data 20+ times. 
Previously, the "per strategy" data was scoped to the for-loop, releasing memory after each iteration.